packages/forklift-console-plugin/src/modules/Providers/ProviderRow.tsx

```diff
@@ -106,11 +106,13 @@ const TextWithIcon = ({ value, Icon }: { value: string; Icon: JSXElementConstruc
 TextWithIcon.displayName = 'TextWithIcon';
 
 const ProviderLink = ({ value, entity, t }: CellProps) => {
-  const isHostProvider = entity.type === 'openshift' && !entity.url;
+  const ownerReferences = entity.object?.metadata?.ownerReferences;
+  const isOwnedByForkliftCOntroller =
```

Comment:
I think it's worth to include this field in the `MergedProvider` interface.  The `entity.object` is an escape hatch to allow integration with legacy code that is reaching deep into the resource structure (without any contract what data is required).